gui/src/components/onboarding/pages/body-proportions/ProportionsResetModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button } from '@/components/commons/Button';
+import { WarningBox } from '@/components/commons/TipBox';
+import { Localized, useLocalization } from '@fluent/react';
+import { BaseModal } from '@/components/commons/BaseModal';
+import ReactModal from 'react-modal';
+
+export function ProportionsResetModal({
+  isOpen = true,
+  onClose,
+  accept,
+  ...props
+}: {
+  /**
+   * Is the parent/sibling component opened?
+   */
+  isOpen: boolean;
+  /**
+   * Function to trigger when the warning hasn't been accepted
+   */
+  onClose: () => void;
+  /**
+   * Function when you press `Reset settings`
+   */
+  accept: () => void;
+} & ReactModal.Props) {
+  const { l10n } = useLocalization();
+
+  return (
+    <BaseModal
+      isOpen={isOpen}
+      shouldCloseOnOverlayClick
+      onRequestClose={onClose}
+      className={props.className}
+      overlayClassName={props.overlayClassName}
+    >
+      <div className="flex w-full h-full flex-col ">
+        <div className="flex flex-col flex-grow items-center gap-3">
+          <Localized id="reset-reset_all_warning" elems={{ b: <b></b> }}>
+            <WarningBox>
+              <b>Warning:</b> This will reset your proportions to the defaults.
+              <br />
+              Are you sure you want to do this?
```

Comment:
Should probably update this text too
```suggestion
              <b>Warning:</b> This will reset your proportions to being just based on your height.
              <br />
              Are you sure you want to do this?
```